marklogic-data-hub-central/ui/src/components/entities/mapping/entity-settings/entity-settings.tsx

```diff
@@ -0,0 +1,229 @@
+      setAdditionalCollections([...props.stepData.additionalCollections]);
+    }
+    if (props.stepData.permissions) {
+      setTargetPermissions(props.stepData.permissions);
+    }
+  };
+
+  const getPayload = () => {
+    return {
+      collections: defaultCollections,
+      additionalCollections: additionalCollections,
+      permissions: targetPermissions,
+    };
+  };
+
+  const isPermissionsValid = () => {
+    if (targetPermissions.trim().length === 0) {
+      setPermissionValidationError(AdvancedSettingsMessages.targetPermissions.incorrectFormat);
+      return false;
+    }
+
+    if (targetPermissions && targetPermissions.trim().length > 0) {
+      let permissionArray = targetPermissions.split(",");
+      for (let i = 0; i < permissionArray.length; i += 2) {
+        let role = permissionArray[i];
+        if (i + 1 >= permissionArray.length || (!role || !role.trim())) {
+          setPermissionValidationError(AdvancedSettingsMessages.targetPermissions.incorrectFormat);
+          return false;
+        }
+        let capability = permissionArray[i + 1];
+        if (!validCapabilities.includes(capability)) {
+          setPermissionValidationError(AdvancedSettingsMessages.targetPermissions.invalidCapabilities);
+          return false;
+        }
+      }
+    }
+    setPermissionValidationError("");
+    return true;
+  };
+
+  const handleAddColl = (value) => {
+    if (value !== " ") {
+      setAdditionalCollections(value.filter((col) => !defaultCollections.includes(col)));
+    }
+  };
+
+  const updateStep = async (payload) => {
+    let stepPayload = Object.assign(props.stepData, payload);
+    await props.updateStep(stepPayload);
+  };
+
+  const handleChange = (event) => {
+    if (event.target.id === "targetPermissions") {
+      setTargetPermissions(event.target.value);
+      if (!targetPermissionsValid && isPermissionsValid()) {
+        setTargetPermissionsValid(true);
+      }
+    }
+  };
+
+  const handleBlur = (event) => {
+    if (event.target.id === "targetPermissions") {
+      setTargetPermissionsValid(isPermissionsValid());
+    }
+  };
+
+  const handleSubmit = async (event: { preventDefault: () => void; }) => {
+    if (event) event.preventDefault();
+    setPopoverVisibilty(false);
+    updateStep(getPayload());
+  };
+
+  const onCancel = () => {
+    setPopoverVisibilty(false);
+    setTargetPermissionsValid(true);
+    setPermissionValidationError(null);
+    getSettings();
+  };
+
+  const togglePopover = () => {
+    popoverVisibility ? setPopoverVisibilty(false) : setPopoverVisibilty(true);
+  };
+
+  const content = (
+    <div id="entity-settings-popover" data-testid="entity-settings-popover" className={styles.entitySettings}>
+      <div className={styles.title}>
+        <p>Advanced Settings: {targetEntityTitle}</p>
+      </div>
+      <div className={styles.text}>
+        <p>Specify additional collections and modify the default permissions for entity instances </p>
```

Comment:
Can we use a single styled p tag for the entire descriptive text instead of using two p's inside a div? You can adjust the width of the p to get the layout you want.